eng/Build.props

```diff
@@ -1,9 +1,19 @@
-<Project>
-  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true' and '$(DotNetBuild)' != 'true'">
+<Project TreatAsLocalProperty="TargetRids">
+  <PropertyGroup>
+    <BuildRid>$([System.Runtime.InteropServices.RuntimeInformation]::RuntimeIdentifier)</BuildRid>
+
+    <!-- Use a colon separate list of RIDs here, like osx-x64:osx-arm64 .
+         It is helpful to allow building the non-aot cli builds on a single CI job -->
+    <TargetRids Condition="'$(TargetRids)' == ''">$(BuildRid)</TargetRids>
+    <SkipManagedBuild Condition="'$(SkipManagedBuild)' == ''">false</SkipManagedBuild>
```

Comment:
NIT: No need to include any of these two here, you can fully remove as nothing is checking that they are set to false. In other words, if they are not set that should be the default, since you are only validating they are not true. Does that make sense?